src/devices/Tlc1543/Tlc1543.cs

```diff
@@ -0,0 +1,289 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device;
+using System.Device.Gpio;
+using System.Collections.Generic;
+
+namespace Iot.Device.Tlc1543
+{
+    /// <summary>
+    /// Add documentation here
+    /// </summary>
+    public class Tlc1543 : IDisposable
+    {
+        #region Members
+
+        /// <summary>
+        /// Channel used between readings to charge up capacitors in ADC
+        /// </summary>
+        public Channel ChargeChannel = Channel.SelfTest512;
+        private readonly int _chipSelect;

```

Comment:
You have to start with the private readonly memebers
So the public property should be after